.github/workflows/ci.yml

```
@@ -57,7 +55,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
+          python-version: "3.11"
```

The tests are meant to be executed with the lowest Python version we support. Otherwise 
```suggestion
          python-version: "3.8"
```